### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by Chipe1**
Added tests for a few functions
Updated README.md

### CODE REVIEW

File: tests/test_csp.py [hunk 1]
```diff
@@ -330,6 +330,15 @@ def test_backtracking_search():
                                order_domain_values=lcv, inference=mac)
 
 
+def test_min_conflicts():
+    random.seed = 'aima-python'
```
Comment (norvig) [inline tests/test_csp.py]: No -- this should be random.seed('aima-python').

File: tests/test_search.py [hunk 2]
```diff
@@ -56,6 +71,33 @@ def test_recursive_best_first_search():
...
+                                       [1, 2, 5, 11]])
+    assert hill_climbing(prob) == (0, 2)
+    prob = PeakFindingProblem((2, 0), [[0, 5, 10, 8],
+                                       [-3, 7, 9, 999],
+                                       [1, 2, 5, 11]])
+    assert hill_climbing(prob) == (1, 3)
+
+
+def test_simulated_annealing():
+    random.seed = 'aima-python'
```
Comment (norvig) [inline tests/test_search.py]: Same here.

**Comment by norvig** [comment]
can you get the tests to pass?

**Comment by Chipe1** [comment]
Sorry about that. I ran py.test on individual files which caused the error to slip past me.

**Pull request merged**